### ISSUE TITLE
Namespace relative linking with periods not working

### ISSUE THREAD

**Issue by jayqi** [glensc/dokuwiki-plugin-pageredirect#19]
Basically I want to redirect `projects:vort2d_kalman_filter` to `projects:vort2d_kalman_filter:start`.

On the page `projects:vort2d_kalman_filter`, a regular internal link `[[.:vort2d_kalman_filter:start]]` properly links to the desired target page. However, the redirect instead actually links to `vort2d_kalman_filter:start` in the root namespace, because it tries to use the URL `mydomain.com/doku.php?id=.vort2d_kalman_filter:start`.

For reference, [here](https://www.dokuwiki.org/namespaces) is Dokuwiki's relative linking syntax.

**Comment by glensc** [comment]
so, for clarity, your page `projects:vort2d_kalman_filter` content is this?:

```
#redirect .:vort2d_kalman_filter:start

[[.:vort2d_kalman_filter:start]] 
```

**Comment by glensc** [comment]
should be fixed with e51fffb
